src/porting_android.cpp

```diff
@@ -88,6 +89,18 @@ android_app* app_global;
 JNIEnv*      jnienv;
 jclass       nativeActivity;
 
+
+void handleAndroidActivityEvents()
+{
+	int ident;
+	int events;
+	struct android_poll_source *source;
+
+	while ( (ident = ALooper_pollOnce(0, NULL, &events, (void**)&source)) >= 0)
+		if (source)
+			source->process(porting::app_global, source);
```

Comment:
Yep, you're probably right... I'll make an update when I read the docs @est31 has referenced